backend/app/services/report_service.py

```diff
@@ -103,10 +103,10 @@ async def get_report_by_id(report_id: UUID) -> Optional[Report]:
     @staticmethod
     async def run(
         timeslot: str,
-        language: str,
+        languages: List[str],
```

Comment:
Maybe we should implement a enum class for languages so we can use it with auto completion
 
```suggestion
        class Language(str, Enum):
          EN = "en"
          DE = "de"
```
and than use it like this: 

```suggestion
        language == Language.DE
```

This isn't only for your code but to the whole project. Just came to me now. Maybe we should do a fix as an own PR